modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -47,6 +58,99 @@ test.afterEach(async ({notificationTemplatesPage, page}) => {
 	}
 });
 
+test('Can use all notification terms in the rich text and check if the content is being persisted', async ({
+	emailNotificationTemplatePage,
+	notificationTemplatesPage,
+	page,
+}) => {
+	await emailNotificationTemplatePage.goto();
+
+	const notificationTemplateName =
+		'Notification Template Name' + getRandomInt();
+
+	await emailNotificationTemplatePage.basicInfoName.fill(
+		notificationTemplateName
+	);
+
+	await emailNotificationTemplatePage.senderEmailAddress.fill(
+		'test@liferay.com'
+	);
+
+	await emailNotificationTemplatePage.senderName.fill('test user');
+
+	await page.getByText('Current User Email Address').click();
+
+	await page.keyboard.press('Tab');
+
+	await page.keyboard.press('Enter');
+
+	await emailNotificationTemplatePage.primaryRecipientUserEmailAddress.click();
+
+	await page.keyboard.press('Control+V');
+
+	await emailNotificationTemplatePage.contentSubject.fill('Content subject');
+
+	await page.getByText('Select an Option').click();
+
+	await page
+		.getByRole('option', {name: objectDefinition.externalReferenceCode})
+		.click();
+
+	const copyButons = await emailNotificationTemplatePage.copyButton.all();
```

Comment:
i think you meant `copyButtons` instead of `copyButons`

Reply:
You are right 😄